rdkit/Chem/SaltRemover.py

```diff
@@ -33,20 +33,61 @@
 
 import os
 import re
+from rdkit.Chem.rdmolfiles import SDMolSupplier
 
 from rdkit import Chem
 from rdkit import RDConfig
 from contextlib import closing
 
+class Format:
+  Smarts, Mol, Smiles = range(3)
```

Comment:
There is a backport of Enum to 2.7, however Greg prefers to keep the dependencies small. 

I prefer strings to get more information when printing.